torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp

```diff
@@ -298,41 +301,19 @@ std::shared_ptr<TransposeParams> getTransposeHeuristics(
     vectorize_factor2 = std::min(vectorize_factor2, tv_vectorize_factor);
   }
 
-  // Try expanding vectorization to contig merged domains
-  auto expanded_vector_word_size1 =
-      scheduler_utils::expandVectorizationToContigMergedDomains(
-          fusion,
-          runtime_info,
-          grouped_inputs_outputs[0],
-          largest1,
-          0,
-          vectorize_factor1);
-  auto expanded_vector_word_size2 =
-      scheduler_utils::expandVectorizationToContigMergedDomains(
-          fusion,
-          runtime_info,
-          grouped_inputs_outputs[1],
-          largest2,
-          0,
-          vectorize_factor2);
-
-  expanded_vector_word_size1 = std::min(
-      static_cast<size_t>(max_unroll_factor), expanded_vector_word_size1);
-  expanded_vector_word_size2 = std::min(
-      static_cast<size_t>(max_unroll_factor), expanded_vector_word_size2);
-
-  vectorize_factor1 = std::max(vectorize_factor1, expanded_vector_word_size1);
-  vectorize_factor2 = std::max(vectorize_factor2, expanded_vector_word_size2);
-
-  params->vectorize_factor1 = vectorize_factor1;
-  params->vectorize_factor2 = vectorize_factor2;
+  params->vectorize_factor1 =
```

Comment:
I don't think this can be re-added. For example,
```
T0[I0, I1, I2, I3] reference
T1[I0, I2, I1, I3]
```
For this case, `T0` and `T1` will be in the same group, but the vectorization factor can not be expanded even with the "virtual innermost dim".

Reply:
Gotcha, yeah it would require having a virtual inner dim that's consistent in the group.